### PR TITLE
Fix 	CleanerThread and read/write test from kafka

### PR DESCRIPTION
**Pull request by fbalicchia**
I play a little around samsa cause i like the idea. thanks if you want check my commit.
Any suggest are wellcome

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -109,6 +109,22 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.3.2</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.10 -->
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.10</artifactId>
+            <version>0.8.2.1</version>
```
Comment (bernd) [inline pom.xml:116]: I guess these dependencies can be in `<scope>test</scope>` because they are only needed for the tests, right?

File: pom.xml [hunk 2]
```diff
@@ -109,6 +109,22 @@
...
+        </dependency>
+        <dependency>
+            <groupId>com.github.joschi</groupId>
+            <artifactId>jadconfig</artifactId>
+            <version>0.13.0</version>
```
Comment (bernd) [inline pom.xml:122]: See above.

File: pom.xml [hunk 3]
```diff
@@ -109,6 +109,22 @@
...
+        </dependency>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>2.9</version>
```
Comment (bernd) [inline pom.xml:128]: See above.

**Review by bernd — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (fbalicchia) [reply pom.xml:116]: Opss, Yes you are right

**Comment by fbalicchia** [comment]
Continue in spare time, as soon as I have something significant' I'll reopen request.
Thanks

**Pull request closed without merging**